365/client/js/router.js

```diff
@@ -36,15 +37,52 @@ Router.onBeforeAction(function() {
     }
 	}
 });
-
+*/
 
 Router.route("main",{
-    path:"/"
+  path:"/",
+  onBeforeAction: function() {
+    if (! Meteor.userId() ) {
+      Session.set('c_login', false);
+      this.redirect('splash');
+    } else {
+      this.next();
+    }
+  }
 })
-Router.route("calendar")
+Router.route("mainPage", function(){
+  this.layout("header");
+  this.render("main");
+});
+Router.route("calendar", function(){
+  this.layout("header");
+  this.render("calendar");
+});
 
 Router.route('/user/:_id', function () {
 
     this.render('user', {data: this.params._id});
 });
 
+Router.route('login', function() { 
+  // this.layout('loginPage');
+  this.layout("splashBanner");
+  this.render('carousel');
+});
+Router.route('splash', function() { 
+  this.layout("splashBanner");
+  this.render('carousel');
+});
+Router.route('about', function() {
+  this.layout("splashBanner");
+  this.render('aboutPage');
+});
+Router.route('team', function() {
+  this.layout("splashBanner");
+  this.render('teamPage');
+});
+Router.route('loginPage', function() {
+  this.layout("splashBanner");
+  // this.render('splash');
+  this.render("loginPage");
+});
```

Comment:
General comments.. can we call the main route something other than 'mainPage'? I thought 'main' was just fine, or how about 'home'. Same thing for other routes like loginPage, the 'Page' suffix is redundant.

Also we need to clean up a lot of old stuff that we're not using anymore. You can either do that on here or I can go through the code later in master and clean it all up, it just has to be done at some point. Sooner rather than later